src/Template/Element/seo_international_targeting.ctp

```diff
@@ -29,19 +29,15 @@ use App\Lib\LanguagesLib;
 use Cake\Core\Configure;
 
 $uiLanguages = Configure::read('UI.languages');
-$currentLang = Configure::read('Config.language');
-$currentUrl = $this->request->here();
+$currentLang = $this->request->getParam('lang');
```

Comment:
Good point, it makes more sense to get the lang from getParam() than Config.language here.